_data/companies/nuance.yml

```
@@ -0,0 +1,7 @@
+---
+name: Nuance Communications [[1]]( https://nuance.com )
```

```suggestion
name: Nuance Communications
```

There's no reference to any advisories on the site, so I'm removing the link.